streams/src/test/java/org/apache/kafka/streams/integration/QueryableStateIntegrationTest.java

```diff
@@ -93,6 +93,7 @@
     private static final long WINDOW_SIZE = TimeUnit.MILLISECONDS.convert(2, TimeUnit.DAYS);
     private static final int NUM_PARTITIONS = 2;
     private static final int NUM_REPLICAS = NUM_BROKERS;
+    private static final int MAX_WAIT_MS = Integer.getInteger("kafka.test.max.wait.ms", 60000);
```

Comment:
In one place there already was 60, in others - 30. One of the cases when I got test failure was timeout, so I've decided to add 30 seconds to see if that helps.

Also now you're able to configure different timeout values for different environments using environmental variables - use shorter timeout for faster PC/server if necessary.